docs/source/about/advanced_usage.rst

```diff
@@ -395,6 +395,32 @@ dwmflush
 
       dwmflush = enabled
 
+preloaded_frames
+-----------------
+
+**Description**
+   This option specifies the number of frames to preload in the image buffer. 
+   Preloading frames improves stream stability at the expense of increased VRAM usage. 
+   If configured with fewer frames, VRAM usage will significantly decrease, but this may cause performance issues.
+   
+   .. Note:: If you're streaming at 4k with HDR enabled, the default setting will reserve roughly 1.8GB of VRAM.
+      It is recommended to reduce the preloaded frames, especially if streaming with those settings and playing modern games.
+
+**Default**
+   ``12``
+
+**Values**
+   An integer value between ``2`` and ``12``, inclusive.
+
+**Example**
+   To set the number of preloaded frames to 6, add the following line to the configuration file:
+
+   .. code-block:: text
+
+      preloaded_frames = 6
+
+   This will preload 6 frames in the image buffer, reducing VRAM usage but potentially decreasing stream stability.
```

Comment:
To remain consistent with the rest of this documentation.

```suggestion
**Range**
   ``2-12``

**Example**
   .. code-block:: text

      preloaded_frames = 12
```

Reply:
I'll be honest, the docs need a complete overhaul because they're not very descriptive at all. But if we want it to be that way, sure I guess.